alexey.zhuravlov/task1/MyHashMap/src/MyHashMap.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Objects;
+
+public class MyHashMap {
+    private MyHashNode[] table;
+    int size;
+
+    public void put(String key, String value) {
+        int hash = Objects.hash(key);
+        int index = hash % (size - 1);
+
+        MyHashNode currentEmptyNode = table[index];
+
+        while (currentEmptyNode != null) {
+            currentEmptyNode = currentEmptyNode.getNext();
+        }
+        currentEmptyNode.setHash(hash);
+        currentEmptyNode.setKey(key);
+        currentEmptyNode.setValue(value);
+        currentEmptyNode.setNext(null);
+
+    }
+
+    public String get(String key) {
+        int hash = Objects.hash(key);
+        int index = hash % (size - 1);
+
+        MyHashNode currentNode = table[index];
+
+        while (currentNode.getHash() != hash) {
+            currentNode = currentNode.getNext();
+        }
+
+        while (!currentNode.getKey().equals(key)) {
+            currentNode = currentNode.getNext();
+        }
+        return currentNode.getValue();
+
+    }
+
+    public void hash(MyHashNode myHashNode) {
+        int hash = Objects.hash(myHashNode.getKey());
+        myHashNode.setHash(hash);
+    }
+
```

Comment:
Can you please add main() to test and demo MyHashMap  work?

I tried to check it. But it's not working

 public static void main(String[] args) {
        MyHashMap myHashMap = new MyHashMap();
        myHashMap.put("test", "test");
    }

Exception in thread "main" java.lang.NullPointerException
	at MyHashMap.put(MyHashMap.java:11)
	at MyHashMap.main(MyHashMap.java:47)